<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackageTab</class>
 <widget class="QWidget" name="PackageTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>608</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeWidget" name="fileBrowser">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <column>
         <property name="text">
          <string>Elemento</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Descrizione</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Percorso completo</string>
         </property>
        </column>
       </widget>
       <widget class="QTreeWidget" name="dependencyBrowser">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16161616</width>
          <height>300</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <column>
         <property name="text">
          <string>Dipendenza</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Da file</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Riga</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Raggiunta</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Versione minima</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QSplitter" name="libTools">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeWidget" name="libraryBrowser">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <column>
         <property name="text">
          <string>Libreria</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Versione</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Percorso</string>
         </property>
        </column>
       </widget>
       <widget class="QTreeWidget" name="packageLibraryViewer">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <column>
         <property name="text">
          <string>Libreria aggiunta</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Percorso da utilizzare</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Versione</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="buttonBox">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="addLibraryButton">
         <property name="text">
          <string>Aggiungi libreria</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="deleteLibraryButton">
         <property name="text">
          <string>Rimuovi libreria</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="infoButton">
         <property name="text">
          <string>Informazioni</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="moveLibraryButton">
         <property name="text">
          <string>Sposta libreria</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Salva</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>80</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exportButton">
         <property name="text">
          <string>Esporta</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionApri_percorso">
   <property name="text">
    <string>Apri percorso</string>
   </property>
  </action>
  <action name="actionInformazioni_libreria">
   <property name="text">
    <string>Informazioni libreria</string>
   </property>
  </action>
  <action name="actionAggiungi_al_pacchetto">
   <property name="text">
    <string>Aggiungi al pacchetto</string>
   </property>
  </action>
  <action name="actionMostra_in_visualizzatore">
   <property name="text">
    <string>Mostra in visualizzatore</string>
   </property>
  </action>
  <action name="actionNascondi">
   <property name="text">
    <string>Nascondi dalla vista</string>
   </property>
  </action>
  <action name="actionIgnora">
   <property name="text">
    <string>Ignora</string>
   </property>
  </action>
  <action name="actionApri_esempi">
   <property name="text">
    <string>Apri esempi</string>
   </property>
  </action>
  <action name="actionApri_keywords">
   <property name="text">
    <string>Apri keywords</string>
   </property>
  </action>
  <action name="actionApri_nel_terminale">
   <property name="text">
    <string>Apri nel terminale</string>
   </property>
  </action>
  <action name="actionImposta_informazioni">
   <property name="text">
    <string>Imposta informazioni</string>
   </property>
  </action>
  <action name="actionApri_documentazione">
   <property name="text">
    <string>Apri documentazione</string>
   </property>
  </action>
  <action name="actionInformazioni">
   <property name="text">
    <string>Informazioni</string>
   </property>
  </action>
  <action name="actionCopia_percorso">
   <property name="text">
    <string>Copia percorso</string>
   </property>
  </action>
  <action name="actionApri_in_visualizzatore">
   <property name="text">
    <string>Apri in visualizzatore</string>
   </property>
  </action>
  <action name="actionApri_con_applicazione_di_sistema">
   <property name="text">
    <string>Apri con applicazione di sistema</string>
   </property>
  </action>
  <action name="actionImposta_flag">
   <property name="text">
    <string>Imposta flag</string>
   </property>
  </action>
  <action name="actionApri_in_terminale">
   <property name="text">
    <string>Apri in terminale</string>
   </property>
  </action>
  <action name="actionImposta_come_file_principale">
   <property name="text">
    <string>Imposta come file principale</string>
   </property>
  </action>
  <action name="actionEscludi_dal_pacchetto">
   <property name="text">
    <string>Escludi dal pacchetto</string>
   </property>
  </action>
  <action name="actionSposta">
   <property name="text">
    <string>Sposta</string>
   </property>
  </action>
  <action name="actionIncolla">
   <property name="text">
    <string>Incolla</string>
   </property>
  </action>
  <action name="actionIncolla_all_interno">
   <property name="text">
    <string>Incolla all' interno</string>
   </property>
  </action>
  <action name="actionEsporta">
   <property name="text">
    <string>Esporta</string>
   </property>
  </action>
  <action name="actionImporta">
   <property name="text">
    <string>Importa</string>
   </property>
  </action>
  <action name="actionAggiorna_vista">
   <property name="text">
    <string>Aggiorna vista</string>
   </property>
  </action>
  <action name="actionElimina">
   <property name="text">
    <string>Elimina</string>
   </property>
  </action>
  <action name="actionElimina_contenuto">
   <property name="text">
    <string>Elimina contenuto</string>
   </property>
  </action>
  <action name="actionCopia">
   <property name="text">
    <string>Copia</string>
   </property>
  </action>
  <action name="actionCopia_contenuto_in_appunti">
   <property name="text">
    <string>Copia contenuto in appunti</string>
   </property>
  </action>
  <action name="actionCopia_contenuto">
   <property name="text">
    <string>Copia contenuto...</string>
   </property>
  </action>
  <action name="actionLista_tag">
   <property name="text">
    <string>Visualizza flag</string>
   </property>
   <property name="toolTip">
    <string>Visualizza tag</string>
   </property>
  </action>
  <action name="actionApri_file_dipendenza">
   <property name="text">
    <string>Apri sorgente</string>
   </property>
   <property name="toolTip">
    <string>Apre il file che richiede questa libreria</string>
   </property>
  </action>
  <action name="actionMostra_libreria">
   <property name="text">
    <string>Mostra libreria</string>
   </property>
   <property name="toolTip">
    <string>Mostra la libreria che soddisfa questa dipendenza</string>
   </property>
  </action>
  <action name="actionMostra_informazioni">
   <property name="text">
    <string>Mostra informazioni</string>
   </property>
   <property name="toolTip">
    <string>Mostra le informazioni della dipendenza</string>
   </property>
  </action>
  <action name="actionVisualizzazione_compatta">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Visualizzazione compatta</string>
   </property>
   <property name="toolTip">
    <string>Imposta la modalità di visualizzazione delle dipendenze</string>
   </property>
  </action>
  <action name="actionNascondi_versioni">
   <property name="text">
    <string>Nascondi versioni</string>
   </property>
  </action>
  <action name="actionApri_file_sorgente_in_ambiente">
   <property name="text">
    <string>Apri file sorgente in ambiente</string>
   </property>
  </action>
  <action name="actionIgnora_dipendenza">
   <property name="text">
    <string>Ignora dipendenza</string>
   </property>
   <property name="toolTip">
    <string>Imposta la dipendenza come da ignorare</string>
   </property>
  </action>
  <action name="actionAbilita_dipendenza">
   <property name="text">
    <string>Abilita dipendenza</string>
   </property>
   <property name="toolTip">
    <string>Abilita il controllo per la dipendenza</string>
   </property>
  </action>
  <action name="actionInformazioniLibreria">
   <property name="text">
    <string>Informazioni</string>
   </property>
  </action>
  <action name="actionApri_percorso_libreria">
   <property name="text">
    <string>Apri percorso</string>
   </property>
   <property name="toolTip">
    <string>Apre il percorso in cui si trova la libreria</string>
   </property>
  </action>
  <action name="actionApri_libreria_terminale">
   <property name="text">
    <string>Apri in terminale</string>
   </property>
   <property name="toolTip">
    <string>Apre il percorso della libreria nel terminale di sistema</string>
   </property>
  </action>
  <action name="actionRimuoviLibreria">
   <property name="text">
    <string>Rimuovi</string>
   </property>
   <property name="toolTip">
    <string>Rimuove la libreria dal pacchetto</string>
   </property>
  </action>
  <action name="actionApriLibInVisualizzatore">
   <property name="text">
    <string>Apri in visualizzatore</string>
   </property>
  </action>
  <action name="actionApri_documentazione_lib">
   <property name="text">
    <string>Apri documentazione</string>
   </property>
  </action>
  <action name="actionApri_esempi_lib">
   <property name="text">
    <string>Apri esempi</string>
   </property>
  </action>
  <action name="actionApri_keywords_lib">
   <property name="text">
    <string>Apri keywords</string>
   </property>
  </action>
  <action name="actionNotALibrary">
   <property name="text">
    <string>Non è una libreria</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
